### PR TITLE
Document minimal neovim version for Copilot

### LINKED ISSUE #21725: Document the minimum Neovim version for Copilot.
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-neovim

### What part(s) of the article would you like to see updated?

Knowing the minimum version of Neovim can avoid wasted time on surprising "Vim version too old" message. Just add simple version number to Prerequisites section so that people can see it immediately instead of digging into the code.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Kixunil**
### Why:

Closes #21725

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Adds information about the minimum Neovim version for Copilot. This is [based on the current code](https://github.com/github/copilot.vim/blob/8ba151a20bc1d7a5c72e592e51bfc925d5bbb837/autoload/copilot/agent.vim#L354-L356)

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@Kixunil Thank you for opening a PR and linking it to your issue! ✨ 

I'll get this triaged for review ⚡

**Review by IcebergVerse — COMMENTED** [review]
kixunil:patch-1

File: content/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-neovim.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ If you use a Neovim, you can view and incorporate suggestions from {% data varia
 
 - To use {% data variables.product.prodname_copilot %} you must have an active {% data variables.product.prodname_copilot %} subscription. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
 
-- To use {% data variables.product.prodname_copilot %} in Neovim you must have Neovim and Node.js version 17 or below installed. For more information, see the [Neovim documentation](https://neovim.io/doc/) and the [Node.js website](https://nodejs.org/en/).
+- To use {% data variables.product.prodname_copilot %} in Neovim you must have Neovim version 0.6+ and Node.js version 17 or below installed. For more information, see the [Neovim documentation](https://neovim.io/doc/) and the [Node.js website](https://nodejs.org/en/).
```
Comment (hubwriter) [inline content/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-neovim.md:24]:
```suggestion
- To use {% data variables.product.prodname_copilot %} in Neovim you must have Neovim version 0.6 or above and Node.js version 17 or below installed. For more information, see the [Neovim documentation](https://neovim.io/doc/) and the [Node.js website](https://nodejs.org/en/).
```

**Review by hubwriter — APPROVED** [review]
@Kixunil - Many thanks for raising the issue and making the docs change in this PR. Much appreciated. 👍 

I'll get this merged/published.

**Pull request merged**